<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>source</class>
 <widget class="QWidget" name="source">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Please choose the installation source for Pardus.

It is recommended to download Pardus from Internet if you do not have Pardus CD. However, if you have Pardus CD or ISO, you can choose those options, too.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="optInternet">
           <property name="text">
            <string>Download Pardus from Internet (recommended, requires Internet)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="optCD">
           <property name="text">
            <string>I have Pardus Live CD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="optISO">
           <property name="text">
            <string>I have Pardus Live CD image (.ISO)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="optUSB">
           <property name="text">
            <string>I have Pardus Live CD contents in my USB memory stick.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
